source/unified-test-format/unified-test-format.rst

```diff
@@ -4072,6 +4080,8 @@ Changelog
 ..
   Please note schema version bumps in changelog entries where applicable.
 
+:2024-01-11: **Schema version 1.18** Allow additional properties on ``kmsProviders`` to test named KMS providers.
```

Comment:
```suggestion
:2024-01-11: **Schema version 1.18.**
             Allow additional properties on ``kmsProviders`` to test named KMS providers.
```

Consistency with previous changelog entries. This message might also change if you switch the `patternProperties`.